src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -977,6 +977,50 @@ int publish_abort(reservation_t& res) {
   return 0;
 }
 
+int get_persistent_queue_by_topic_name(const DoutPrefixProvider* dpp, librados::IoCtx& rados_ioctx, std::string& topic_name,
```

Comment:
you probably want the topic name to be `const std::string&`

Reply:
Ack!